src/Lykke.Service.ClientSearch/Startup.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;

```

Comment:
Please compare the class with the reference implementation and change it accordingly. https://github.com/LykkeCity/lykke.dotnettemplates/blob/0b506d6d6326a558c7107c40c4f78f7ae1f5bac0/Lykke.Service.LykkeService/src/Lykke.Service.LykkeService/Startup.cs#L1

Reply:
Startup.cs code renewed.
New code is taken from service template.